### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#913] [Galaxy Trivia] [answered]
🟦👑🟥⬇️
🟦🟥🏰🟦
🟩🟨⭐🟥

**Comment by peterbaccaro** [comment, marked as answer]
1. LIECHTENSTEIN
2. CAMBODIA
3. SENEGAL

![image](https://github.com/ParanoidUser/yolo/assets/74666079/c59819b3-9523-45fa-99f5-d65ec0c377a9)

**Comment by raeef1001** [comment]
LIECHTENSTEIN
CAMBODIA
SENEGAL
